src/app/components/nav/simple-header/simple-header.component.ts

```diff
@@ -0,0 +1,149 @@
+import {ChangeDetectorRef, Component, ElementRef, HostListener, Inject, OnDestroy, OnInit, ViewChild} from '@angular/core';
+import {GlobalService} from '../../../services/global.service';
+import {ActivatedRoute, Router} from '@angular/router';
+import {AuthService} from '../../../services/auth.service';
+import {ApiService} from '../../../services/api.service';
+import {DOCUMENT} from '@angular/common';
+
+@Component({
+  selector: 'app-simple-header',
+  templateUrl: './simple-header.component.html',
+  styleUrls: ['./simple-header.component.scss']
+})
+export class SimpleHeaderComponent implements OnInit, OnDestroy {
```

Comment:
Why its name is simple header ?

Reply:
I named it as per used in angularJS project.